examples/cluster/plot_coin_ward_segmentation.py

```diff
@@ -9,44 +9,55 @@
 
 """
 
+# %%
```

Comment:
This marker makes the rendering weird with some of the font being in bold.
You can either revert or make the rendering maybe with some RST items, as you prefer.